src/chrome/content/rules/Fastmail-User.FM.xml

```diff
@@ -0,0 +1,24 @@
+<!--
+          For other Fastmail coverage, see Fastmail.xml 
+
+
+		  Rewrite rule for user.fm illustrated:
+
+          http://robm.fastmail.fm.user.fm/articles/arthurs_walk.html
+                 ^^^^^^^^^^^^^^^^         ^^^^^^^^^^^^^^^^^^^^^^^^^^
+                        $1                             $2
+
+      => https://user.fm/www.robm.fastmail.fm/articles/arthurs_walk.html
+                         ^^^^^^^^^^^^^^^^^^^^ ^^^^^^^^^^^^^^^^^^^^^^^^^^
+                                  $1                      $2
+-->
+<ruleset name="Fastmail User Websites">
+	<target host="user.fm" />
+	<target host="*.user.fm" />
+		<test url="http://robm.fastmail.fm.user.fm/articles/arthurs_walk.html" />
```

Comment:
Are all targets subdomains of `fm.user.fm`? If so, it would probably be better to put the wildcard on this domain.

Reply:
No. That is from the tld of the hosted website. If www.example.com was hosted there, it would also be available as http://www.example.com.user.fm and https://user.fm/www.example.com 

A few examples from Sublist3r:

www.rjj.id.au.user.fm
www.hairtodyefor.biz.user.fm
www.3karpfen.ch.user.fm
www.bilikfamily.com.user.fm
telescope.cslwin.fastmail.com.user.fm
antelopeweb.fmail.co.uk.user.fm